dask/array/percentile.py

```diff
@@ -68,15 +68,15 @@ def merge_percentiles(finalq, qs, vals, Ns, interpolation='lower'):
 
     finalq : numpy.array
         Percentiles to compute (must use same scale as ``qs``).
-    qs : sequence of numpy.arrays
+    qs : sequence of :function:`numpy.array`s
```

Comment:
Should probably be `:class:`numpy.ndarray`, or just say "sequence of numpy arrays.